heatmap/heatmap.go

```diff
@@ -6,6 +6,9 @@ import (
 	"github.com/K-Phoen/grabana/errors"
 	"github.com/K-Phoen/grabana/heatmap/axis"
 	"github.com/K-Phoen/grabana/links"
+
+	// "github.com/K-Phoen/grabana/target/cloudwatch"
```

Comment:
@fetchkarthikkasarapu perhaps this line here was left in the branch by accident?